packages/backend/src/graphql/schema.graphql

```diff
@@ -1247,6 +1255,8 @@ input UpdateIncomingPaymentInput {
   id: ID!
   "The new metadata object to save for the incoming payment. It will overwrite any existing metadata."
   metadata: JSONObject!
+  "The tenant UUID associated with the incoming payment. If not provided, it will be obtained from the signature."
+  tenantId: String
```

Comment:
Given that this is for an existing resource, we don't need to have this on the Input. We will use the signature tenant id to check if they can update the payment.